src/locales/pt/users.json

```
@@ -0,0 +1,109 @@
+{
+  "list": {
+    "title": "Gerenciamento de Usuários",
+    "actions": {
+      "createUser": "Criar Usuário"
+    }
+  },
+  "feedbacks": {
+    "activateUserSuccess": {
+      "title": "Usuário Ativado",
+      "description": "Usuário \"{{login}}\" ativado com sucesso"
+    },
+    "deactivateUserSuccess": {
+      "title": "Usuário Desativado",
+      "description": "Usuário \"{{login}}\" desativado com sucesso"
+    },
+    "activateUserError": {
+      "title": "Falha na Ativação",
+      "description": "Falha ao ativar o usuário \"{{login}}\""
+    },
+    "deactivateUserError": {
+      "title": "Falha na Desativação",
+      "description": "Falha ao desativar o usuário \"{{login}}\""
+    },
+    "deleteUserSuccess": {
+      "title": "Usuário Excluído",
+      "description": "Usuário \"{{login}}\" excluído com sucesso"
+    },
+    "deleteUserError": {
+      "title": "Falha na Exclusão",
+      "description": "Falha ao remover o usuário \"{{login}}\""
+    },
+    "loadingUserError": {
+      "title": "Falha ao carregar usuários"
+    }
+  },
+  "data": {
+    "login": {
+      "alreadyUsed": "Nome de usuário já utilizado",
+      "label": "Nome de Usuário",
+      "required": "Nome de usuário é obrigatório",
+      "tooShort": "Nome de usuário é muito curto (mín. {{min}} caracteres)",
+      "tooLong": "Nome de usuário é muito longo (máx. {{max}} caracteres)",
+      "invalid": "Nome de usuário é inválido, não use caracteres especiais"
+    },
+    "email": {
+      "label": "E-mail",
+      "alreadyUsed": "E-mail já utilizado",
+      "required": "E-mail é obrigatório",
+      "invalid": "E-mail é inválido",
+      "tooShort": "E-mail é muito curto (mín. {{min}} caracteres)",
+      "tooLong": "E-mail é muito longo (máx. {{max}} caracteres)"
```

There is diff here with the `account.json`. Both `email` and `e-mail` are used in Portuguese, you should choose one to use...

```suggestion
      "required": "O email é obrigatório",
      "invalid": "O email é inválido",
      "tooShort": "O email é muito curto (mín. {{min}} caracteres)",
      "tooLong": "O email é muito longo (máx. {{max}} caracteres)"
```